coinmarketcap/src/endpoint/marketcap.ts

```diff
@@ -1,37 +1,43 @@
 import { Requester, Validator } from '@chainlink/external-adapter'
-import { ExecuteWithConfig, Config } from '@chainlink/types'
+import { ExecuteWithConfig, Config, ResponsePayload } from '@chainlink/types'
+import { getSymbolData } from './price'
 
-export const NAME = 'globalmarketcap'
+export const NAME = 'marketcap'
 
-const marketcapParams = {
-  market: ['market', 'to', 'quote'],
+const priceParams = {
+  symbol: ['base', 'from', 'coin', 'sym', 'symbol'],
+  quote: ['quote', 'to', 'market', 'convert'],
 }
 
 export const execute: ExecuteWithConfig<Config> = async (request, config) => {
```

Comment:
Same thoughts as other adapters that a lot of code could be reduced by using a `path` in price.